ratis-examples/src/main/bin/common.sh

```diff
@@ -38,7 +38,7 @@ fi
 
 echo "Found ${ARTIFACT}"
 
-QUORUM_OPTS="--peers n0:localhost:6000,n1:localhost:6001,n2:localhost:6002"
+QUORUM_OPTS="--peers n0:localhost:6000:6010:6020:6030,n1:localhost:6001:6011:6021:6031,n2:localhost:6002:6012:6022:6032"
```

Comment:
+1

Reply:
> The port setting should be optional, i.e. when there is only a single port, it will be used for both client and admin.

It is optional.  I changed it here to prove it works, but I'm OK reverting it.